pkg/metrics/metrics.go

```diff
@@ -80,12 +81,14 @@ func (m *Reporter) generateMetrics() []*dto.MetricFamily {
 	}
 
 	desiredFamily := generateGaugeFamily("nodereaper_instance_group_desired_size", "Desired number of nodes in the instance group")
+	remainingFamily := generateGaugeFamily("nodereaper_instance_group_remaining_to_delete", "Remaining number of nodes to delete")
 	statesFamily := generateGaugeFamily("nodereaper_instance_group_state", "The number of nodes in a particular state of deletion")
```

Comment:
If I understand it right, this PR doesn't seem necessary. 
nodereaper_instance_group_state{state!="dont_want_delete"}
should gave you all the node that's in the states of being deleted. count() over it would give you the size as calculated in this PR. Here is an example query:
count(nodereaper_instance_group_state{state!="dont_want_delete"}) by (az, group) >0

Reply:
Agreed, thanks!

Reply:
I misunderstood what nodereaper_instance_group_state does, I thought it's for each individual node but it's actually a count for number of nodes in each state. 
But that doesn't make it harder (actually a bit simpler):
sum(nodereaper_instance_group_state{state!="dont_want_delete"} by (az, group)
is equivalent to this new metric nodereaper_instance_group_state, right?
The delta of sum is sum of delta, so:
sum(delta(nodereaper_instance_group_state{state!="dont_want_delete"}[5m])) by (az, group) is what you want.

Reply:
Agreed, thanks!